### DISCUSSION TITLE
Welcome to Complete-Campus-Enrollment-System Discussions!

### DISCUSSION THREAD

**Discussion by JehanKandy** [JehanKandy/Complete-Campus-Enrollment-System#5] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JehanKandy** [comment]
Start to asking Questions, share Ideas and join with projects

**Reply by learnkandy123** [reply]
good evening

**Reply by JehanKandy** [reply]
good evening

**Reply by JehanKandy** [reply]
How can I help you

**Reply by jkandysl** [reply]
good evening

**Reply by JehanKandy** [reply]
good evening

**Reply by JehanKandy** [reply]
How can I help you
